tests/homeScreening.spec.js

```diff
@@ -0,0 +1,70 @@
+import { jest } from '@jest/globals';
+import getTenScreenings from '../src/getTenScreenings.js';
+
+describe('getRecentReviews()', () => {
+  beforeEach(() => {
+    jest.useFakeTimers();
+  });
+
+  afterEach(() => {
+    jest.clearAllTimers();
+  });
+
+  test('Only show screenings for the next five days', async () => {
+    jest.setSystemTime(new Date('2024-02-02T13:37:00.000Z'));
+
+    const getTenScreeningsAdapter = {
+      loadAllScreenings: async () => [
+        mockTime({ start_time: '2024-02-02T19:00:00.000Z' }),
+        mockTime({ start_time: '2024-02-03T19:00:00.000Z' }),
+        mockTime({ start_time: '2024-02-10T19:00:00.000Z' }),
+      ],
+    };
+
+    const data = await getTenScreenings(getTenScreeningsAdapter);
+    expect(data).toHaveLength(2);
+  });
+
+  test('Only show screenings for the next five days', async () => {
+    jest.setSystemTime(new Date('2024-02-02T13:37:00.000Z'));
+
+    const getTenScreeningsAdapter = {
+      loadAllScreenings: async () => [
+        mockTime({ start_time: '2024-02-02T19:00:00.000Z' }),
+        mockTime({ start_time: '2024-02-03T19:00:00.000Z' }),
+        mockTime({ start_time: '2024-02-02T19:00:00.000Z' }),
+        mockTime({ start_time: '2024-02-03T19:00:00.000Z' }),
+        mockTime({ start_time: '2024-02-02T19:00:00.000Z' }),
+        mockTime({ start_time: '2024-02-03T19:00:00.000Z' }),
+        mockTime({ start_time: '2024-02-02T19:00:00.000Z' }),
+        mockTime({ start_time: '2024-02-03T19:00:00.000Z' }),
+        mockTime({ start_time: '2024-02-02T19:00:00.000Z' }),
+        mockTime({ start_time: '2024-02-03T19:00:00.000Z' }),
+        mockTime({ start_time: '2024-02-02T19:00:00.000Z' }),
+        mockTime({ start_time: '2024-02-03T19:00:00.000Z' }),
+        mockTime({ start_time: '2024-02-10T19:00:00.000Z' }),
+      ],
+    };
+
+    const data = await getTenScreenings(getTenScreeningsAdapter);
+    expect(data).toHaveLength(10);
+  });
+
+  function mockTime(attributes) {
+    return {
+      id: 201,
+      time: attributes.start_time,
```

Comment:
If all you're using the `attributes` argument for is the time, maybe you could have just used a single argument for the time (and not an object), which would have made the code easier to use and read.